libs/pcre/Makefile

```diff
@@ -61,6 +61,18 @@ define Package/libpcrecpp
   DEPENDS:=+libpcre $(CXX_DEPENDS)
 endef
 
+include $(INCLUDE_DIR)/host-build.mk
+
+HOST_CONFIGURE_ARGS += \
+	--enable-utf8 \
+	--enable-unicode-properties \
+	--enable-pcre16 \
+	--with-match-limit-recursion=16000 \
```

Comment:
Why?

Reply:
I'm not an author but from the documentation it looks like it should limit stack usage to ~8Mb
"As a very rough rule of thumb, you should reckon on about 500 bytes per recursion. Thus, if you want to limit your stack usage to 8Mb, you should set the limit at 16000 recursions. A 64Mb stack, on the other hand, can support around 128000 recursions."

https://www.pcre.org/original/doc/html/pcrestack.html
https://www.pcre.org/original/doc/html/pcrebuild.html#SEC14

Reply:
I'd mark this as resolved @neheb ?